auxx/cardano-sl-auxx.cabal

```diff
@@ -4,8 +4,9 @@ synopsis:            Cardano SL - Auxx
 description:         Cardano SL - Auxx
 license:             MIT
 license-file:        LICENSE
-author:              Serokell
-maintainer:          hi@serokell.io
+homepage:            https://github.com/input-output-hk/cardano-sl
+author:              IOHK
```

Comment:
When the code has two authors, isn't it customary to keep both of them in the `author` section? :wink: